docs/installation/prepare/rpi-setup.md

```diff
@@ -172,26 +172,31 @@ sudo apt upgrade
 ## Enable Communication Protocols
 
 If you are using hardware that requires I2C, SPI, serial, or one-wire protocols to communicate with your Pi, you will need to enable them using `raspi-config`.
+These protocols are required to support certain hardware, such as analog-to-digital converters (ADCs), accelerometers, and sensors.
```

Comment:
[opt] To me, the current way sounds almost redundant/odd because it's essentially saying twice that it's required. I think I like the second suggestion  best because it keeps the lists in separate sentences whilst introducing the why before the how, though it makes "them" ambiguous.
```suggestion
Certain pieces of hardware, such as analog-to-digital converters (ADCs), accelerometers, and sensors, communicate with your Pi using I2C, SPI, serial, or one-wire protocols.
If you are using hardware that requires any of these protocols, you need to enable them using `raspi-config`:
```
```suggestion
Certain hardware requires I2C, SPI, serial, or one-wire protocols to communicate with your Pi.
If you are using hardware such as an analog-to-digital converter (ADC), accelerometer, or sensor that requires any of these protocols, you need to enable them using `raspi-config`:
```

Reply:
I slightly like the first way better